recipe/meta.yaml

```diff
@@ -19,14 +19,20 @@ build:
 requirements:
     build:
         - perl
-        - gcc  # [linux]
+        - gcc  # [not win]
```

Comment:
I was told for other recipes to include this for future-proofness, in case we do enable Windows some day, but happy to remove if people prefer.